Examples/Tests/ohm_solver_ion_beam_instability/inputs_test_1d_ohm_solver_ion_beam_picmi.py

```diff
@@ -347,8 +343,8 @@ def text_diag(self):
 
         status_dict = {
             "step": step,
-            "nplive beam ions": self.ion_container_wrapper.nps,
-            "nplive ions": self.beam_ion_container_wrapper.nps,
+            "nplive beam ions": self.ion_container.size,
+            "nplive ions": self.beam_ion_container.size,
```

Comment:
@roelof-groenewald I realize now that the original code already mismatches the dict keys to the values on the RHS. Shall I fix on the go?
```suggestion
            "nplive ions": self.ion_container.size,
            "nplive beam ions": self.beam_ion_container.size,
```